### DISCUSSION TITLE
Many releases with localization errors

### DISCUSSION THREAD

**Discussion by alexmlt1971** [iina/iina#4551] [General]
After the release of 1.2.0, many useful features have been added. But several releases have already been released, new functions are not localized in the settings. In particular, HDR. Are you working on fixing these bugs? Thank you.

**Comment by alexmlt1971** [comment]
Registered on Crowdin today. Made changes to the localization of Russian and Ukrainian. When will the changes be released? 
Found an error. "Enable HDR" settings in the Codec section has a translation in Crowdin, but always in English in the application.
Thank you.

**Comment by low-batt** [comment]
Thank you for pointing out the error with IINA not using localized strings for the  `Enable HDR mode` setting. I have entered issue #4558 for that defect.

Broken localization like this is considered a high priority defect. One of the reasons for the quick release of 1.3.3 was due to broken localization. Sorry this one was missed. If you notice any other failures to use localizations, please bring it to our attention.

As for when changes made in Crowdin appear in IINA. We always make sure all Crowdin translations are pulled into GitHub as a part of preparing a release.

As for when a version of IINA will next be released, I'm the newbie junior developer and can't answer that question. The 3<sup>rd</sup> party libraries IINA is dependent upon were upgraded for 1.3.2. Issues are starting to be posted regarding regressions that are due to upgrading those libraries. It would be nice if those could be fixed in the next release as well as some of IINA's own issues.
